invenio_rdm_migrator/streams/streams.py

```diff
@@ -10,6 +10,26 @@
 
 from datetime import datetime
 
+from ..extract import Extract
+from ..transform import Transform
+
+
+class NoExtract(Extract):
```

Comment:
On the one hand we could simply have the base class do that, on the other hand is better if it is a conscious decision.